src/util/Shared.js

```diff
@@ -123,6 +123,15 @@ export class Shared {
 
     return parseFloat(resolution) * mpu * inchesPerMeter * dpi;
   }
+
+  /**
+   * Removes duplicated forward slashes as well as trailing slash
+   * and returns normalized URL string
+   * @param {*} url
+   */
+  static sanitizeUrl = (url) => {
+    return url.replace(/([^:]\/)\/+/g, '$1').replace(/\/+$/, '');
+  }
```

Comment:
Not blocking, but this method would be a great candidate for the `base-util`.

Reply:
Good point! Backported it in https://github.com/terrestris/base-util/pull/351